package.json

```diff
@@ -20,7 +20,7 @@
     "build-forms": "lerna exec --scope=@talend/react-forms -- yarn run prepublish",
     "build-icons": "lerna exec --scope=@talend/icons -- yarn run prepublish && lerna exec --scope=@talend/react-components -- yarn test -- -u",
     "build-theme": "lerna exec --scope=@talend/bootstrap-theme -- yarn run prepublish",
-    "test": "lerna exec --concurrency 1 -- TZ=Europe/Paris yarn test",
+    "test": "lerna exec --concurrency 1 -- TZ=Europe/Paris yarn test --detectOpenHandles --forceExit",
```

Comment:
I don't really understand this option. Can you explain why you added it ?

Reply:
it was to debug the CI which hang with jest. I have finally used --runInBand which fix the issue.
I have used --silent also which speed up the global execution which is now under 10 minutes :D